### PR TITLE
Cleaned up repos and added unit tests for them.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java [hunk 1]
```diff
@@ -68,6 +67,9 @@ public long create(final Exam exam) {
     }
 
     private void batchCreateExamClaims(final List<ExamClaim> examClaims, final long examId) {
+        //TODO: is it correct?
```
Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java]: Should the repo assume that the service does the validation and if it is given an ICA exam with no claims - that is alright?

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java [hunk 2]
```diff
@@ -100,6 +102,9 @@ private long create(final StudentExamAttributes studentExamAttributes) {
     }
 
     private void batchCreateExamItems(final List<ExamItem> examItems, final long examId) {
+        //TODO: is it correct?
+        if (examItems.isEmpty()) return;
```
Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java]: Same as above but for items?

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java (hunk 1 above)
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java]: I"d vote yes, but i can see both sides of it.

Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java]: Ok. Then I will assume this is right.

**Pull request merged**